File: PATTERNS/Pattern33/Pattern33.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter the input n where row = 2*n - 1:', (n) => {
    for (let i = 1; i <= 2 * n - 1; i++) {
        if(i==n){
            for(let j=0;j<2*n-1;j++){
                process.stdout.write("*");
            }
            process.stdout.write("\n");
            continue;
        }
        for (let k = 0; k <n-1; k++) {
            process.stdout.write(" ");
        }
        process.stdout.write("* ");
        process.stdout.write("\n");
    }
    rl.close();
});

File: PATTERNS/Pattern33/Pattern33.java
import java.util.Scanner;



public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the input n where row = 2*n - 1: ");
        int n = scanner.nextInt();
        int flag = 1;
        for (int i = 1; i <= 2 * n - 1; i++) {
            if (i == n) {
                flag = 2 * n - 1;
            }
            for (int k = 1; k <= n - flag; k++) {
                System.out.print(" ");
            }
            for (int j = 1; j <= flag; j++) {
                System.out.print("+");
            }
            flag = 1;
            System.out.println();
        }
    }
}
